bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/utils/EspConfigParser.java

```diff
@@ -0,0 +1,65 @@
+package com.espressif.idf.debug.gdbjtag.openocd.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+import com.espressif.idf.core.logging.Logger;
+
+public class EspConfigParser
+{
+	@SuppressWarnings("unchecked")
+	public ArrayList<String> getTargets()
+	{
+
+		ArrayList<String> targets = new ArrayList<String>();
+		InputStream is = EspConfigParser.class.getResourceAsStream("tcl_esp-config.json"); // $NON-NLS-1$
```

Comment:
We can replace this with the $OPENOCD_SCRIPTTS/esp-config.json once it's available

Reply:
replaced. Path to json now is:
https://github.com/espressif/idf-eclipse-plugin/blob/878166be511ae39c09d60ded505b9dd07c5b40cb/bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/utils/EspConfigParser.java#L30-L31